integration_tests/CMakeLists.txt

```diff
@@ -267,7 +290,7 @@ macro(RUN)
     if ((NOT DISABLE_FAST) AND (NOT RUN_NOFAST))
         set(RUN_EXTRA_ARGS ${RUN_EXTRA_ARGS} --fast)
         set(RUN_NAME "${RUN_NAME}_FAST")
-        list(REMOVE_ITEM RUN_LABELS cpython cpython_py) # remove cpython, cpython_py from --fast test
+        list(REMOVE_ITEM RUN_LABELS cpython sympy cpython_py) # remove cpython, sympy, cpython_py from --fast test
```

Comment:
I guess the above would be `cpython_sym` inplace of `sympy`?

Reply:
Yes that's correct !

Reply:
>Actually sympy essentially was replaced by c_sym, so maybe we could include that too ?

I think we should not include it. `cpython_sym` means the test runs using `CPython` and I think `CPython` does not support the `--fast` flag. Therefore tests for the backend `cpython_sym` should not be run with `--fast` flag. 

`c_sym` uses `lpython's` `c` backend. `lpython` supports `--fast` flag. Therefore if the given test file supports/compiles-with `--fast`, we should test both the versions non-fast and `--fast`.

Reply:
Yes , that's a good argument . It does make sense .